crates/profile/models/security-structures/src/roles_matrices_structures/matrices/builder/error.rs

```diff
@@ -17,8 +17,8 @@ pub enum MatrixRolesInCombinationBasicViolation {
     #[error("The factor source was not found in any role")]
     FactorSourceNotFoundInAnyRole,
 
-    #[error("The number of days until auto confirm must be greater than zero")]
-    NumberOfDaysUntilAutoConfirmMustBeGreaterThanZero,
+    #[error("The number of days until timed confirm is callable must be greater than zero")]
+    NumberOfDaysUntilTimeBasedConfirmationMustBeGreaterThanZero,
```

Comment:
Breaking Profile format change number 2 - @sergiupuhalschi-rdx  / @danvleju-rdx  / @matiasbzurovski might get affected in host